### PR TITLE
Tuple module

### PR DESCRIPTION
**Pull request by lpil**
Using Elm's Tuple module as inspiration.

### CODE REVIEW

File: src/tuple.alp [hunk 1]
```diff
@@ -0,0 +1,82 @@
+{-| Some helpers for working with 2-tuples.
```
Comment (j14159) [inline src/tuple.alp:1]: Out of curiousity, I'm not familiar with the `{-|` comment syntax.  Where is it from?

**Comment by j14159** [comment]
Looks good, thanks for this!

File: src/tuple.alp (hunk 1 above)
Comment (lpil) [reply src/tuple.alp:1]: Ah! It's the syntax for a documentation comment in haskell and elm, meaning that the comment will be used when generating pretty html documentation for the module. I wrote it habitually.

Shall I continue to use this out shall I remove the pipe to make it a normal comment?

Comment (j14159) [reply src/tuple.alp:1]: Short answer:
Doesn't bother me to leave it in atm but let's discuss it further under Alpaca :)

Longer answer:
I'm vaguely of the opinion that any comment preceding an AST node should be automatically attached to that node as documentation for output so I'm not sure if we need a special form (this ties into formatters and alternate ways of storing source).  But!  I think this is a good thing to discuss, especially here:  https://github.com/alpaca-lang/alpaca/issues/134

**Pull request merged**